### ISSUE TITLE
SET application_name sends $1 to the server

### ISSUE THREAD

**Issue by trasch** [vapor/postgres-nio#272]
### Describe the bug

I want to set the application name on the connection right after connecting to the server.

I'm using:
```
let connection = try await PostgresConnection.connect(...)
let applicationName = "something"
try await connection.query("SET application_name=\(applicationName)", logger: logger)
```

The server receives:
```
ERROR:  syntax error at or near "$1" at character 22
STATEMENT:  SET application_name=$1
```
... i.e. just a placeholder, which is evidentally wrong.

It does work when I rewrite the query by using `PostgresQuery` directly:
```
try await connection.query(PostgresQuery(stringLiteral: "SET application_name='\(applicationName)'"), logger: logger)
```

### To Reproduce

Connect to a PostgreSQL server and send the query above.

### Expected behavior

The connection should have an application name (but I guess that this is a problem for any SET query)

### Environment

* postgres-nio version: 1.9.0
* OS version: MacOS 12.3 / Xcode 13.3 / Swift 5.6
* PostgreSQL server version: 11.1

**Comment by 0xTim** [comment]
This was solved in #258

String interpolation uses binding by default, you can write 
```swift
let connection = try await PostgresConnection.connect(...)
let applicationName = "something"
try await connection.query("SET application_name=\(unescaped: applicationName)", logger: logger)
```

Hope that helps!

**Comment by trasch** [comment]
@0xTim 
Almost, I needed to add single quotes (`'`), otherwise the query will hang (why?):

```
try await connection.query("SET application_name='\(unescaped: applicationName)'", logger: logger)
```

I would recommend to add this to the README since this wasn't evident to me, even after quite a bit of searching 🙂
